services/redis-config-map.yaml

```diff
@@ -0,0 +1,839 @@
+    # publisher can produce them).
+    #
+    # The limit can be set differently for the three different classes of clients:
+    #
+    # normal -> normal clients including MONITOR clients
+    # slave  -> slave clients
+    # pubsub -> clients subscribed to at least one pubsub channel or pattern
+    #
+    # The syntax of every client-output-buffer-limit directive is the following:
+    #
+    # client-output-buffer-limit <class> <hard limit> <soft limit> <soft seconds>
+    #
+    # A client is immediately disconnected once the hard limit is reached, or if
+    # the soft limit is reached and remains reached for the specified number of
+    # seconds (continuously).
+    # So for instance if the hard limit is 32 megabytes and the soft limit is
+    # 16 megabytes / 10 seconds, the client will get disconnected immediately
+    # if the size of the output buffers reach 32 megabytes, but will also get
+    # disconnected if the client reaches 16 megabytes and continuously overcomes
+    # the limit for 10 seconds.
+    #
+    # By default normal clients are not limited because they don't receive data
+    # without asking (in a push way), but just after a request, so only
+    # asynchronous clients may create a scenario where data is requested faster
+    # than it can read.
+    #
+    # Instead there is a default limit for pubsub and slave clients, since
+    # subscribers and slaves receive data in a push fashion.
+    #
+    # Both the hard or the soft limit can be disabled by setting them to zero.
+    client-output-buffer-limit normal 0 0 0
+    client-output-buffer-limit slave 256mb 64mb 60
+    client-output-buffer-limit pubsub 32mb 8mb 60
+
+    # Redis calls an internal function to perform many background tasks, like
+    # closing connections of clients in timeout, purging expired keys that are
+    # never requested, and so forth.
+    #
+    # Not all tasks are performed with the same frequency, but Redis checks for
+    # tasks to perform according to the specified "hz" value.
+    #
+    # By default "hz" is set to 10. Raising the value will use more CPU when
+    # Redis is idle, but at the same time will make Redis more responsive when
+    # there are many keys expiring at the same time, and timeouts may be
+    # handled with more precision.
+    #
+    # The range is between 1 and 500, however a value over 100 is usually not
+    # a good idea. Most users should use the default of 10 and raise this up to
+    # 100 only in environments where very low latency is required.
+    hz 10
+
+    # When a child rewrites the AOF file, if the following option is enabled
+    # the file will be fsync-ed every 32 MB of data generated. This is useful
+    # in order to commit the file to the disk more incrementally and avoid
+    # big latency spikes.
+    aof-rewrite-incremental-fsync yes
+kind: ConfigMap
+metadata:
+  creationTimestamp: 2017-08-09T23:23:31Z
+  name: sd-redis-master-conf
+  namespace: cd-screwdriver-k8s
```

Comment:
Maybe use something a little more generic as the namespace example

Reply:
updated